src/docfx/learn/Achievement.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Docs.Build
+{
+    internal class Achievement
+    {
+        public SourceInfo<string> Uid { get; set; } = new SourceInfo<string>("");
```

Comment:
prefer _init only properties_
```suggestion
        public SourceInfo<string> Uid { get; init; } = new SourceInfo<string>("");
```

Reply:
OK